packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -47,12 +46,18 @@ export interface TreeNodeRendererOwnProps {
    * Actions for tree item.
    */
   actions?: Array<(node: PresentationHierarchyNode) => TreeItemAction>;
+  /**
+   * Used to display elements between expander and label.
+   * Elements are arranged from left to right, with the last array element positioned on the right.
```

Comment:
Looks like this comment need to be updated after return type was changed to `ReactNode`

Reply:
Removed the part about element arrangement.